pyccel/codegen/printing/ccode.py

```diff
@@ -457,18 +480,20 @@ def _print_Print(self, expr):
             if isinstance(f, ValuedVariable):
                 if f.name == 'sep'      :   sep = str(f.value)
                 elif f.name == 'end'    :   end = str(f.value)
+            elif isinstance(f, FunctionCall) and isinstance(f.dtype, NativeTuple):
```

Comment:
Are (inhomogenous) tuples supported now? Or just for the print stage?

Reply:
No this is only for the print stage .